nucliadb/nucliadb/reader/api/models.py

```diff
@@ -55,8 +55,8 @@
 class ResourceField(BaseModel):
     field_type: FieldTypeName
     field_id: str
-    value: ValueType
-    extracted: ExtractedDataType
+    value: ValueType = None
```

Comment:
This should be `Optional[ValueType]` instead no?

Reply:
`ValueType` is defined above as `Optional[Union[...]]` so it's already an Optional